kong/init.lua

```diff
@@ -1431,7 +1429,7 @@ function Kong.serve_cluster_listener(options)
 
   kong_global.set_phase(kong, PHASES.cluster_listener)
 
-  return clustering.handle_cp_websocket()
+  return kong.clustering.child:handle_cp_websocket()
```

Comment:
nitpick: should this be `kong.clustering:handle_cp_websocket()` (and we propogate
to `child` in clustering/init.lua)? As there's no other places accessing `child` directly.